Makefile

```diff
@@ -59,10 +60,11 @@ run: generate fmt vet ## Run against the configured Kubernetes cluster in ~/.kub
 deploy: manifests ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 	kustomize build config/default | kubectl apply -f -
 
-
 .PHONY: manifests
 manifests: ## Generate manifests e.g. CRD, RBAC etc.
 	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
+	cp -f ./config/rbac/rbac*.yaml ./config/ci/rbac/
+	cp -f ./config/manager/manager*.yaml ./config/ci/manager/
```

Comment:
Why copy these to a separate directory here and not copy them to a temporary directory as part of the ci target workflow?

Either that or possibly add `./config/ci` to .gitignore to avoid checking in duplicate versions of these files.

Reply:
+1 On a .gitignore rule

Reply:
yup, will add them to .gitignore, thanks